cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -506,3 +502,50 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                        indirect=True)
+def test_migration_special_char_name(request, appliance, v2v_providers, host_creds, conversion_tags,
+                                    form_data_vm_obj_single_datastore):
```

Comment:
Question : test name says " test_migration_special_char_name" . 
Can you please add a doc string explaining what and how is the test testing special chars name ?

Reply:
Added.